src/cameraproxy.h

```diff
@@ -104,6 +105,7 @@ public Q_SLOTS:
 private:
     std::shared_ptr<libcamera::CameraManager> m_cameraManager;
     std::shared_ptr<libcamera::Camera> m_currentCamera;
+    Settings *m_settings;
```

Comment:
Maybe here should default to nullptr, and check not null when used.  Would be ore robust.